### DISCUSSION TITLE
同步mysql大表任务会卡主无法发送数据也不报错

### DISCUSSION THREAD

**Discussion by qianchutao** [apache/flink-cdc#1440] [Q&A]
用flinkCDC 2.2版本同步msyql数据，initial模式，在同步比较大的表时，全量快照阶段任务会一直卡主，日志没有异常，但也无法发送数据到kafka，请问有什么参数可以调优吗？
目前的配置：
taskManger：12g 
jobmanager： 2g 
slot数量：4  
并行度：4
chunk-size：50000
fetch-size：2048

**Comment by gezhiwei8899** [comment]
我们也是类似的。 观察下jobmanager chunk 条数呢？贴给我看看
